platform/darwin/src/MGLComputedShapeSource.mm

```diff
@@ -155,4 +153,8 @@ - (void)reloadData {
     self.rawSource->reload();
 }
 
+- (mbgl::style::CustomVectorSource*) rawSource {
+    return _pendingSource.get();
```

Comment:
I dont think this will work, because _pendingSource gets set to null when the source is added to the map.